<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1604</width>
    <height>1207</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1604</width>
    <height>1207</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1604</width>
    <height>1207</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1604</width>
     <height>1207</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1604</width>
     <height>1207</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget{border-image: url(:/background/resource/background1.png);}</string>
   </property>
   <widget class="QLabel" name="open_door_fire">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>140</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/tool/resource/door_open.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="open_door_ice">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>140</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/tool/resource/door_open.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="diamond1">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>100</y>
      <width>48</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/diamond/resource/blue.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="diamond2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>48</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/diamond/resource/red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="diamond3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>48</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/diamond/resource/red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="diamond4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>550</y>
      <width>48</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/diamond/resource/red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="diamond5">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>1030</y>
      <width>48</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/diamond/resource/red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="diamond6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>48</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/diamond/resource/blue.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="diamond7">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>580</y>
      <width>48</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/diamond/resource/blue.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="diamond8">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>1030</y>
      <width>48</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/diamond/resource/blue.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="trampoline">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>800</y>
      <width>111</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/tool/resource/trampoline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fire_pool">
    <property name="geometry">
     <rect>
      <x>745</x>
      <y>1133</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/tool/resource/fire_pool.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ice_pool">
    <property name="geometry">
     <rect>
      <x>1065</x>
      <y>1131</y>
      <width>201</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/tool/resource/ice_pool.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="poisonous_pool">
    <property name="geometry">
     <rect>
      <x>988</x>
      <y>892</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/tool/resource/poisonous_pool.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dead_blue">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>1070</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/tool/resource/ghost.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dead_red">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>1070</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/tool/resource/ghost.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dead_green">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>830</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/tool/resource/ghost.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="box">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>590</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/tool/resource/box.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="portal2">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>152</y>
      <width>41</width>
      <height>145</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/tool/resource/portal.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="portal1">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>320</y>
      <width>41</width>
      <height>145</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/tool/resource/portal.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="button2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>277</y>
      <width>64</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/tool/resource/button.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="button1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>440</y>
      <width>64</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/tool/resource/button.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fireboy">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>890</y>
      <width>120</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/character/resource/fire_boy_front_1.0.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="icegirl">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>1050</y>
      <width>120</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/character/resource/ice_girl_front_1.0.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1604</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
